lib/cli.mjs

```diff
@@ -206,6 +206,10 @@ const helpText = `
       --help, -h
 
         Show the help.
+
+      --compute-edit-info, --cei
+
+        Compute edit information for fixable problems.
```

Comment:
This should be placed after l84/`--fix`.
i.e. --version and --help must remain last

Reply:
I've fixed it.
[Move help message position](https://github.com/stylelint/stylelint/pull/8473/commits/b5920a83428c9452f99115300efd8ed52ff8b5ac)